src/schedule/2020-12-28-style-community-calendar.md

```diff
@@ -7,10 +7,14 @@ Join us as we add CSS to our [Eleventy](https://11ty.dev)-based [community calen
 
 Check out [our pull request](https://github.com/LunchDevCommunity/community-calendar/pull/10)!
 
+<div data-responsive-youtube--container>
+<iframe src="https://www.youtube.com/embed/M4moKmALiMQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
 ## Shared Links
 
-* [Eleventy's docs for `addPassthroughCopy`](https://www.11ty.dev/docs/copy/)
-* [CSS Tricks on system font stacks](https://css-tricks.com/snippets/css/system-font-stack/)
-* [Michael Knepprath's homepage](https://mknepprath.com/)
-* [Josh W. Comeau's full-bleed tutorial](https://www.joshwcomeau.com/css/full-bleed/)
-* [Andy Bell's modern CSS reset](https://piccalil.li/blog/a-modern-css-reset/)
\ No newline at end of file
+- [Eleventy's docs for `addPassthroughCopy`](https://www.11ty.dev/docs/copy/)
```

Comment:
Can you help me understand the move from asterisks for bullets to dashes? They're equivalent and produce the same markup, so I'm curious if this is just a preference thing.

Reply:
oh crap! I'm sorry. i didn't even notice this change. i don't have a preference.

i have this twitch where i run the `format` command in VS code (basically every time i make a change).
tbh, i usually write them with asterisks as well and then prettier changes them 😆

question tho, `prettier` should be running as a pre-commit hook, shouldn't it? if so, shouldn't these be standardized at that point?

Reply:
Ah, auto-format. I'm fine with these dashes, no worries!

I think you're right, re: Prettier.